modules/phase_field/test/tests/phase_field_kernels/ADAllenCahn.i

```diff
@@ -0,0 +1,75 @@
+#
+# Test the parsed function free enery Allen-Cahn Bulk kernel
+#
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 10
+  ny = 10
+  xmax = 12
+  ymax = 12
+  elem_type = QUAD4
+[]
+
+[Variables]
+  [./eta]
+    order = FIRST
+    family = LAGRANGE
+    [./InitialCondition]
+      type = SmoothCircleIC
+      x1 = 0.0
+      y1 = 0.0
+      radius = 6.0
+      invalue = 0.9
+      outvalue = 0.1
+      int_width = 3.0
+    [../]
+  [../]
+[]
+
+[Kernels]
+  [./detadt]
+    type = ADTimeDerivative
+    variable = eta
+  [../]
+
+  [./ACBulk]
+    type = ADAllenCahn
+    variable = eta
+    f_name = F
+  [../]
+
+  [./ACInterface]
+    type = ADACInterface
+    variable = eta
+    kappa_name = 1
+    variable_L = false
```

Comment:
do we need to make a test with L_variable = true?

Reply:
We probably should :-(

Reply:
I added the suggested test (also for the non-AD version)

Reply:
👍